src/Chronos.php

```diff
@@ -182,6 +196,112 @@ public static function hasTestNow(): bool
         return static::$testNow !== null;
     }
 
+    /**
+     * Sets the time.
+     *
+     * @param int $hours Hours of the time
+     * @param int $minutes Minutes of the time
+     * @param int $seconds Seconds of the time
+     * @param int $microseconds Microseconds of the time
+     * @return static
+     */
+    public function setTime(int $hours, int $minutes, int $seconds = 0, int $microseconds = 0): static
+    {
+        $new = clone $this;
+        $new->native = $new->native->setTime($hours, $minutes, $seconds, $microseconds);
+
+        return $new;
+    }
+
+    /**
+     * Creates a new instance with date modified according to DateTimeImmutable::modifier().
+     *
+     * @param string $modifier Date modifier
+     * @return static
+     * @throws \InvalidArgumentException
+     */
```

Comment:
A `@see` or `@link` tag to the PHP manual for the modifier string format would be nice.